tests/test_modeling_tf_common.py

```diff
@@ -799,23 +799,27 @@ def recursive_check(tuple_object, dict_object):
             dict_inputs = self._prepare_for_class(inputs_dict, model_class)
             check_equivalence(model, tuple_inputs, dict_inputs, {"output_hidden_states": True})
 
-            tuple_inputs = self._prepare_for_class(inputs_dict, model_class)
-            dict_inputs = self._prepare_for_class(inputs_dict, model_class)
-            check_equivalence(model, tuple_inputs, dict_inputs, {"output_attentions": True})
+            # Pure conv models (such as ConvNeXt) don't have `output_attentions`.
+            if config.output_attentions:
+                tuple_inputs = self._prepare_for_class(inputs_dict, model_class)
+                dict_inputs = self._prepare_for_class(inputs_dict, model_class)
+                check_equivalence(model, tuple_inputs, dict_inputs, {"output_attentions": True})
```

Comment:
The core test should not be touched. Instead the test should be overwritten for models that do not have attentions. Note that his change removes the testing for all models, as `config.output_attentions` is always `False` at this stage.

Reply:
@sgugger I have moved the respective equivalence tests. See if this works now. Once the tests are resolved I will get to what @gante and @ydshieh are [proposing regarding](https://github.com/huggingface/transformers/pull/15750#issuecomment-1048928740) transpositions.